dom.go

```diff
@@ -750,6 +751,10 @@ func (d document) DocumentURI() string {
 	return d.Get("documentURI").String()
 }
 
+func (d document) ExecCommand(name string, showDefaultUI bool, valueArgument string) bool {
+	return d.Call("execCommand", name, showDefaultUI, valueArgument).Bool()
+}
```

Comment:
All I can tell is this use case worked for me:

`dom.GetWindow().Document().ExecCommand("copy", false, "") // document.execCommand('copy');`